owncloud-android-library/build.gradle

```diff
@@ -0,0 +1,26 @@
+apply plugin: 'com.android.library'
+
+dependencies {
+    api 'com.squareup.okhttp3:okhttp:3.12.0'
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.3.11"
+    implementation 'com.gitlab.ownclouders:dav4android:oc_support'
+}
+
+android {
+    compileSdkVersion 28
+
+    defaultConfig {
+        minSdkVersion 14
+        targetSdkVersion 28
```

Comment:
Upgrading the sdk version is not always a trivial task and we do it carefully, with a specific issue to test everything and ensure nothing is broken, so I would keep 26 in `compileSdkVersion` and  `targetSdkVersion` here and in sample client. Android P migration will be faced separately in https://github.com/owncloud/android/issues/2283

Reply:
It violets my principe "never go back" and don't introduce Lint warnings intentionally 
<img width="1144" alt="image" src="https://user-images.githubusercontent.com/3314607/49997288-76024080-ff91-11e8-9c01-4669589a7a8f.png">

But I want to do the shit done, so it's 26 again